external/common_functions.sh

```diff
@@ -178,6 +181,36 @@ function getProperty() {
     echo $PROP_VALUE
 }
 
+function set_external_custom_test_info(){
+    test=$1
+    for current_test in ${supported_external_custom_tests}
```

Comment:
This for loop requires for each new third party app we will need to update the source code, which means it can't be used directly by tweeter api. For external_custom target we actually don't need to check the test target. This can be simplified as :
 
```
function set_external_custom_test_info(){
                PROPERTY_FILE=external_custom/test.properties
                GITHUB_URL=$(getProperty "github_url")
                github_url="${EXTERNAL_CUSTOM_REPO}"
                echo "The value of the EXTERNAL_CUSTOM_REPO in common_functions.sh is ${EXTERNAL_CUSTOM_REPO}"
                echo "The test name is ${test}"
                script="test.sh"
                test_command="${EXTERNAL_TEST_CMD}"
                echo "The value of the EXTERNAL_TEST_CMD in common_functions.sh is ${EXTERNAL_TEST_CMD}"
                test_results="testResults"
                tag_version="${EXTERNAL_REPO_BRANCH}"
                echo "The value of the EXTERNAL_REPO_BRANCH in common_functions.sh is ${EXTERNAL_REPO_BRANCH}"
                environment_variable="MODE=\"java\""
                debian_packages="git maven"
                debianslim_packages="${debian_packages}"
                ubuntu_packages="${debian_packages}"
                alpine_packages="git maven"
                centos_packages="git maven"
                clefos_packages="${centos_packages}"
                ubi_packages="git maven"
                ubi_minimal_packages="${ubi_packages}"
}

This can be updated when you do the cleanup to remove unnecessary echoes in the following PRs.

Reply:
Yes this can be updated. I will do so in the next PR. Thank you!